pkg/client/tp_client.go

```diff
@@ -22,6 +22,14 @@ type TPClient struct {
 	*RESTClient
 }
 
+func (c *TPClient) GetJwtToken(hydraToken string) (string, error) {
+	panic("implement me")
```

Comment:
Nit: Although I am not sure when if ever this can be hit, a more explainatory error will be useful.